test/contracts/core/tokens/rebalancingSetToken.spec.ts

```diff
@@ -755,7 +756,8 @@ contract('RebalancingSetToken', accounts => {
         const minimumBid = biddingParameters[0];
         await rebalanceAuctionModule.bid.sendTransactionAsync(
           rebalancingSetToken.address,
-          minimumBid
+          minimumBid,
+          false
```

Comment:
This block between `756-760` is repeated over and over again. Consider adding a rebalancing helper method that also defaults the partial fill parameter to false